src/storage.rs

```diff
@@ -215,6 +215,13 @@ fn convert_domain_tag_to_ddb_tag(tag: Tag) -> HashMap<String, AttributeValue> {
     let mut v: AttributeValue = Default::default();
     v.bool = Some(tag.canary);
     map.insert("canary".to_owned(), v);
+
+    if tag.load_balancing_weight.is_some() {
```

Comment:
(nit) `mut` and `map` are unnecessary

```rust
if let Some(weight) = tag.load_balancing_weight {
  let v = AttributeValue { n: Some(weight.to_string()), ..Default::default() };
  map.insert("load_balancing_weight".to_owned(), v);
}
```

Reply:
Thanks! fixed https://github.com/cookpad/sds/pull/2/commits/7cfc312cdedc18767ccf6e1ea740a18e20b8f042